xl2times/__main__.py

```diff
@@ -119,12 +119,11 @@ def convert_xl_to_times(
         transforms.include_tables_source,
         transforms.merge_tables,
         transforms.complete_processes,
-        transforms.apply_more_fixups,
         transforms.process_units,
-        transforms.process_years,
         transforms.complete_commodity_groups,
-        transforms.process_uc_wildcards,
         transforms.process_wildcards,
+        transforms.apply_transforms,
```

Comment:
Should we call this `process_transform_tables` or `apply_transform_tables`? In this context where everything above and below is a transform, apply transforms is a bit confusing.